react-basic-practice/src/themes/foundations/colors.ts

```diff
@@ -1,12 +1,13 @@
-export const colors = {
+const colors = {
   riverbed: "#464F60",
   mirage: "#171C26",
   paleSky: "#687182",
   shuttleGray: "#5A6376",
-  gullGray: "#A1A9B8",
```

Comment:
remove this color but I see you still use it in `react-basic-practice/src/components/Search/index.tsx`. Pls double check